docs/topics/testing/advanced.txt

```diff
@@ -67,6 +67,35 @@ The following is a simple unit test using the request factory::
             response = MyView.as_view()(request)
             self.assertEqual(response.status_code, 200)
 
+.. _topics-testing-advanced-allowed-hosts:
+
+Tests and ``ALLOWED_HOSTS``
+===========================
+
+.. versionadded:: 1.11
+
+If your application includes unit tests with custom host names, those tests
```

Comment:
That was how I had it setup originally, actually -- but someone (I forget who) recommended this route instead. I actually prefer this pattern. Testing with custom hosts is definitely the exception, not the rule, so I think it makes sense to use this method.